### PR TITLE
add FortiSwitch 3032E and related modules and PSU

### PR DESCRIPTION
**Pull request by kifeo**
adds:
the FortiSwitch 3032E
40G QSFP+ module
100G QSFP28 module
Fortiswitch 3032E PSU

Also images of the switch front/back

### CODE REVIEW

File: device-types/Fortinet/FS-3032E.yaml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  - name: port28
+    position: '28'
+  - name: port29
+    position: '29'
+  - name: port30
+    position: '30'
+  - name: port31
+    position: '31'
+  - name: port32
+    position: '32'
```
Comment (danner26) [inline device-types/Fortinet/FS-3032E.yaml:83]: Are these network interfaces?? If so, they should be under `interfaces` not `module-bays`

**Review by danner26 — CHANGES_REQUESTED** [review]

File: device-types/Fortinet/FS-3032E.yaml (hunk 1 above)
Comment (kifeo) [reply device-types/Fortinet/FS-3032E.yaml:83]: these ports are QSFP ports,  that can be filled with 40G or 100G transceivers (provided in the PR), or with splitters (4x10G or 4x25G), to be added after in another PR

**Comment by danner26** [comment]
@kifeo 
It looks like this PR hasn't been worked on even though we reviewed it. We are going to close it for now, but feel free to re-open the PR if you get a chance to work on it. Thank you.

**Comment by kifeo** [comment]
@danner26 I've added a comment to your comment, and resolved the conversation. I'm not sure what more is expected here ? Thanks

File: device-types/Fortinet/FS-3032E.yaml (hunk 1 above)
Comment (kifeo) [reply device-types/Fortinet/FS-3032E.yaml:83]: Hi @danner26 ,

The ports are module slots that contains transceivers, or splitters. I'm willing to push for other Fortiswitches equipments only if this one will be accepted already. Did you have a look at my previous comment ? Thanks

**Pull request closed without merging**